src/Microsoft.CodeAnalysis.FxCopAnalyzers/RulesMissingDocumentation.md

```diff
@@ -2,5 +2,5 @@
 
 Rule ID | Missing Help Link | Title |
 --------|-------------------|-------|
-CA1416 | https://docs.microsoft.com/visualstudio/code-quality/ca1416 | Validate platform compatibility |
+CA1416 | <https://docs.microsoft.com/visualstudio/code-quality/ca1416> | Validate platform compatibility |
```

Comment:
This is auto-generated file by tool, please revert and change the tool if required: https://github.com/dotnet/roslyn-analyzers/blob/dc649547a3d3a80d4e8caffc7bdd31cf3e70cb92/src/Tools/GenerateDocumentationAndConfigFiles/Program.cs#L467